lib/msf/core/exploit/http/client.rb

```diff
@@ -187,6 +188,10 @@ def connect(opts={})
       'DigestAuthIIS'          => datastore['DigestAuthIIS']
     )
 
+    if datastore['HttpHeaders']
```

Comment:
This needs to update to `HttpRawHeaders` too.

Reply:
Thanks.

Reply:
The code is no longer a lie.